### ISSUE TITLE
Setting acceptsKeyboardFocus should act as the main toggle switch to enable/disable keyboard focusibility

### ISSUE THREAD

**Issue by lamxdoan** [microsoft/react-native-windows#4733]
### Environment

1. `react-native -v`: 
react-native-cli: 2.0.1
react-native: 0.60.6
2. `react-native run-windows --info`:
  System:
    OS: Windows 10 10.0.18363
    CPU: (12) x64 Intel(R) Xeon(R) CPU E5-1650 v4 @ 3.60GHz
    Memory: 13.89 GB / 31.92 GB
  Binaries:
    Node: 11.3.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.16.0 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.4.1 - C:\Program Files\nodejs\npm.CMD
  npmPackages:
    react: 16.8.6 => 16.8.6 
    react-native: 0.60.6 => 0.60.6 
    react-native-windows: 0.60.0-vnext.155 => 0.60.0-vnext.155
  Installed UWP SDKs:
    10.0.16299.0
    10.0.17134.0
    10.0.17749.0
    10.0.17763.0
    10.0.18362.0
3. `reg query "HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock"`
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock
    AllowDevelopmentWithoutDevLicense    REG_DWORD    0x1
    AllowAllTrustedApps    REG_DWORD    0x1

- Target Platform Version(s):     10.0.18362.0

- Target Device(s): Desktop

- Visual Studio Version: 2017

- Build Configuration: Debug, Release

### Steps to Reproduce

Currently if acceptsKeyboardFocus is set to false, it doesn't always result in a control that is not focusable. Depending on other accessibility props, a ContentControl could still be created and have keyboard focus.

### Expected Behavior
Setting acceptsKeyboardFocus should act as the main toggle switch and false should always mean the control is not keyboard focusable.

### Actual Behavior
Setting acceptsKeyboardFocus doesn't act as the main toggle switch for keyboard focus.

### Additional context

This issue was mentioned at the keyboard focus design meeting.

**Comment by lamxdoan** [comment]
@kmelmon

**Comment by chrisglein** [comment]
Duplicate of #4703
@lamxdoan make sure that the direction there reflects the discussion we had. Plan is to remove the -1 stuff.
